### DISCUSSION TITLE
Problem with UI lib

### DISCUSSION THREAD

**Discussion by rootoor-dev** [vlang/v#20909] [Installation Issues]
I got this error when I tried to run **ved IDE**. The installation is correct for I followed all the steps. My installation path is "~/.vars/app/v".
V runs successfully but it is impossible to run any code containing UI lib. By the way, the UI lib has been installed successfully too.
My OS is kali linux purple already updated. It is based on debian and debian is at his 12th version.
I need help to use v ui lib in a real project. Thank you
![photo_2024-02-26_10-42-55](https://github.com/vlang/v/assets/87162205/720b86ff-44fa-48cf-ade8-a104671933da)

**Comment by LukyGuyLucky** [comment]
Try
 v -cc gcc -shared .
could help
Good luck!
